McJon/puplets
# Download from production theme

### Goal
Initial theme download

### Testing Instructions
- Visit [Homepage](https://nb4xzu7fhzcfav7f-29911810107.shopifypreview.com) make sure site builds without error
- Ensure that all files from the production theme have been downloaded

### Code Request Quality Checklist


### Reviewers
@sarahvr92
